alws/constants.py

```diff
@@ -17,6 +17,20 @@ def is_finished(cls, status):
         return status not in (cls.IDLE, cls.STARTED)
 
 
+class SignTaskStatus(enum.IntEnum):
+
+    IDLE = 0
+    PENDING = 1
+    STARTED = 2
+    COMPLETED = 3
+    FAILED = 4
+    EXCLUDED = 5
+
+    @classmethod
+    def is_finished(cls, status):
+        return status not in (cls.IDLE, cls.COMPLETED)
```

Comment:
I guess FAILED is also a finish in some way. Are you sure we need EXCLUDED ? I don't catch the case.

Reply:
EXCLUDED is not needed for sign because we will not sign anything for excluded items